dockerfiles/build_manylinux_x86_64.Dockerfile

```diff
@@ -95,22 +104,24 @@ RUN which gcc && gcc --version && \
 # in git 2.35.3
 RUN git config --global --add safe.directory '*'
 
-######## vcpkg ########
-# vcpkg is used to install OpenMPI and can be dropped once the latter
-# is vendored into TheRock.
-WORKDIR /opt
-ENV VCPKG_HASH="3f5ad7be7693ce6ac5599ddb7cc24f260b9d44f9"
-COPY install_vcpkg.sh ./
-RUN yum install -y zip unzip
-RUN ./install_vcpkg.sh "${VCPKG_HASH}"
```

Comment:
We most likely also want to get rid of vcpkg and these scripts if they are no longer needed.

Reply:
Please delete the orphan `install_vcpkg.sh` as it is no longer used with your changes.